### ISSUE TITLE
Deploy demo site to official Hoaxy domain

### ISSUE THREAD

**Issue by glciampaglia** [IUNetSci/hoaxy-botometer#248]
The switch must occur right before the press release, due to embargo with journalists.

**Comment by filmenczer** [comment]
@benabus Will you please also add Fakey (fakey.iuni.iu.edu) to the OSoMe tools page, when ready?

**Comment by filmenczer** [comment]
We also need to double check that the "top" articles under the search bar are updated automatically. (@mihaivavram says that should happen without intervention.)

**Comment by glciampaglia** [comment]
I have ran manually the cron job script of the news API. It runs without errors, and produces an updated file with the latest news. But the website does not update accordingly, even after force reloading. @mihaivavram, @benabus  any idea?

**Comment by benabus** [comment]
Correct.  The Demo site is not looking for the updated file on Carl.  When launched, it will.


Ben Serrette
Web Developer
Indiana University Network Science Institute

**Comment by filmenczer** [comment]
Press Release is *this Thursday*. @benabus and @mihaivavram please deploy site on Wed around 3pm (that is when reporters will be given embargoed early access). Thanks!
